src/query-builder/insert-query-builder.ts

```diff
@@ -339,6 +339,65 @@ export class InsertQueryBuilder<DB, TB extends keyof DB, O>
     })
   }
 
+  /**
+   * This can be used to add any additional SQL to the front of the query __after__ the `insert` keyword.
```

Comment:
This is not aligned with the example, as the additional SQL is not added **right after the keyword**. Confusing.

Reply:
Will remove it!